base/src/main/java/io/spine/base/Tests.java

```diff
@@ -71,7 +73,7 @@
      *         the system property explicitly.
      */
     @Override
-    protected boolean enabled() {
+    public boolean enabled() {
```

Comment:
This was a design purpose to prohibit invoking this method from outside the package. This is why they were not enums.